concordium-sdk/src/main/java/com/concordium/sdk/types/UInt8.java

```diff
@@ -0,0 +1,57 @@
+package com.concordium.sdk.types;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.val;
+
+import java.io.IOException;
+
+@EqualsAndHashCode
+@Getter
+@JsonSerialize(using = UInt8.UInt8Serializer.class)
+public class UInt8 {
```

Comment:
... an unsigned byte

Reply:
It was needed for a smart contract parameters in the examples and we decided to implement it in the sdk as we have UInt16-64.

Reply:
Something like this "An unsigned byte, used in the smart contract examples."? Not quite sure i understand what needs to be here.

Reply:
So this is a bit better. 
But 1) If it's only used in examples, should it then be here at all? Why not just in the examples then? 
2) This is very specific to a smart contract right? It's not a type that is exposed by the node itself. It could just have exposed an `i8` instead.
3) Is it really needed with this boxed type? Why not just in the smart contract examples use `byte` and then convert to unsigned when doing the serialization?

Reply:
We added it to the SDK for "completeness", as we already have UInt16-32-64. It is only used for these smart contract examples tho, so i believe it would make sense to move it to the examples.

If raw `byte` is used then custom serialization would have to be defined in several places as `UInt8` is used in a lot of different parameters. I believe it is more intuitive, and less error prone, to do it in one place?

Reply:
Removed this class and added `UInt8ByteArrayJsonSerializer` to the examples to handle the serialization of the now used `byte[]`